lib/client/Client.js

```diff
@@ -397,6 +397,56 @@ export default class Client {
   }
 
   /**
+   * Updates properties of a {@link Message}.<br />
+   * This will issue a {@link Client#setMessages} JMAP request under the hoods, passing the correct options.
+   *
+   * @param id {String} The id of the {@link Message} to update.
+   * @param options {Object} The options of the target message to be updated.
+   * @param options.mailboxIds {String[]} The identifiers of the new mailboxes for the message.
+   * @param options.isFlagged {Boolean} This corresponds whether the message is flagged or not
+   * @param options.isUnread {Boolean} This corresponds whether the message has been yet read or not
+   * @param options.isAnswered {Boolean} This corresponds whether the message has been yet replied or not
+   *
+   * @return {Promise} A {@link Promise} that eventually resolves to nothing if the message was updated successfully.
```

Comment:
What do you think about returning the new Message ? It'd allow to do code like 

    message
      .update({isFlagged: true})
      .then(updatedMessage => updateView(updatedMessage))